src/OpenTK.Windowing.Common/Input/WindowIcon.cs

```diff
@@ -0,0 +1,36 @@
+﻿//
+// WindowIcon.cs
+//
+// Copyright (C) 2018 OpenTK
+//
+// This software may be modified and distributed under the terms
+// of the MIT license. See the LICENSE file for details.
+//
+
+namespace OpenToolkit.Windowing.Common.Input
```

Comment:
Why is this in Input?

Reply:
Because the original WindowIcon was.

Reply:
I think this should be changed to just Windowing.Common, if you agree.

Reply:
That seems logical.